website/content/en/docs/reference/configuration/sources/mqtt.md

```diff
@@ -0,0 +1,14 @@
+---
+title: MQTT
+description: Receive observability event data from an [MQTT](https://mqtt.org) broker
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏Since we're being specific to log data, I think we should keep that consistent here.